doc/endpoint/security.md

```diff
@@ -49,6 +49,34 @@ will show you a password prompt.
 Optional and multiple authentication inputs have some additional rules as to how hey map to documentation, see the
 ["Authentication inputs and security requirements"](../docs/openapi.md) section in the OpenAPI docs for details.
 
+## Limiting request body length
+
+*Supported backends*: 
+Feature enabled only for Netty-based servers. More backends will be added in the near future.
+
+Individual endpoints can be annotated with content length limit:
+
+```scala mdoc:compile-only
+import sttp.tapir._
+import sttp.tapir.server.model.EndpointExtensions._
+
+val limitedEndpoint = endpoint.maxRequestBodyLength(maxBytes = 163484L)
+```
+
+The `EndpointsExtensions` utility is available in `tapir-server` core module. If you can't depend on it where your endpoint 
+definitions are located, you can directly put an attribute:
```

Comment:
that's not really useful as it also requires a dependency on server-core ;)

Reply:
Yeah, maybe that case class was in `tapir-core` previously, I don't remember now ;) But I think we should keep it all in server-core, not core, wdyt?

Reply:
yeah let's start with that :)